sys/log/stub/include/log/log.h

```diff
@@ -43,38 +43,62 @@ static inline int
 log_register(const char *name, struct log *log, const struct log_handler *h,
              void *arg, uint8_t level)
 {
+    (void)name;
+    (void)log;
+    (void)log_handler;
```

Comment:
@fjmolinas maybe you don't have access to travis output to see why build failed.

```c
(void)log_handler;
```
should be
```c
(void)h;
```

Reply:
Face palm... copy pasta, it seems I did a poor compile test, sorry about that.